reconcile/jenkins_job_builds_cleaner.py

```diff
@@ -86,7 +86,7 @@ def run(dry_run):
         token = instance["token"]
         instance_name = instance["name"]
         jenkins = JenkinsApi.init_jenkins_from_secret(
-            secret_reader, token, ssl_verify=False
+            secret_reader, token, ssl_verify=True
```

Comment:
nit: this should be configurable via app-interface but can be done in another PR